docs/world api.md

```diff
@@ -140,7 +140,8 @@ Other classifications include
 * `filler`: a regular item or trash item
 * `useful`: generally quite useful, but not required for anything logical
 * `trap`: negative impact on the player
-* `skip_balancing`: add to `progression` to skip balancing; e.g. currency or tokens
+* `skip_balancing`: can be combined with `progression` to skip progression balancing
+* `progression_skip_balancing`: a progression item that skips progression balancing; e.g. currency or tokens
```

Comment:
I'd probably denote this is a combination of `progression` and `skip_balancing`.

Then maybe reword `skip_balancing` to just be something like: denotes this item is not moved earlier during progression balancing.